src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //initialize variables
+        int listSize = 0;
+        Task[] list = new Task[100];
+        boolean isRunning = true;
+
+        //get initial response
+        Scanner in = new Scanner(System.in);
+        IntroductoryMessage();
+
+        //run the chatbot
+        runIkaros(listSize, list, isRunning, in);
+        goodbyeMessage();
+    }
+
+    private static void runIkaros(int listSize, Task[] list, boolean isRunning, Scanner in) {
+        String response;
+        String[] command;
+        while (isRunning) {
+            response = in.nextLine();
+            lineBreak();
+            command = response.split(" ", 10);
+            switch (command[0]) {
+            case "echo":
+                echo();
+                break;
+            case "todo":
+                ToDo task = new ToDo(response.substring(5));
+                listSize = addToList(task, listSize, list, response);
+                break;
+            case "list":
+                printList(list, listSize);
+                lineBreak();
+                break;
+            case "done":
+                done(list, response);
+                break;
+            case "deadline":
+                String date = response.substring(response.indexOf("/") + 4);
+                Deadlines work = new Deadlines(response.substring(9, response.indexOf("/") - 1),
+                        date);
+                listSize = addToList(work, listSize, list, response);
+                break;
+            case "event":
+                String timing = response.substring(response.indexOf("/") + 4);
+                Event event = new Event(response.substring(6, response.indexOf("/") - 1), timing);
+                listSize = addToList(event, listSize, list, response);
+                break;
+            case "bye":
+                isRunning = false;
+                break;
```

Comment:
Can consider abstracting the code inside each case statement into individual functions